app/furniture/marketplace/order.rb

```diff
@@ -52,5 +52,126 @@ def marketplace_name
     def price_total
       [product_total, delivery_fee, tax_total].compact.sum
     end
+
+    def square_client
+      environment = ENV.fetch("SQUARE_ENV")
+      access_token = marketplace.square_access_token
+
+      @square_client ||= Square::Client.new(access_token:, environment:)
+    end
+
+    def send_to_square_seller_dashboard(stripe_balance_transaction)
+      square_create_order_response = create_square_order
+      square_order_id = square_create_order_response.body.order[:id]
+      create_square_order_payment(square_order_id)
+
+      # TODO: What to return here?
+      true
+    end
+
+    def create_square_order
+      square_create_order_body = build_square_create_order_body(marketplace)
+      @square_client.orders.create_order(body: square_create_order_body)
+    end
+
+    # NOTE: Square requires that orders are paid in order to show up in the Seller
+    # Dashboard
+    def create_square_order_payment(square_order_id)
+      square_location_id = marketplace.settings["square_location_id"]
+      space_id = marketplace.space.id
+      square_create_payment_body = build_square_create_order_payment_body(
+        square_order_id,
+        square_location_id,
+        space_id,
+        # TODO: what should be canonical pricing data here: stripe_balance_transaction or order data?
```

Comment:
I would expect this to be the `sellers_share` of the order; not the full balance or order amount.